scalafmt-tests/src/test/scala/org/scalafmt/util/HasTests.scala

```diff
@@ -72,10 +74,14 @@ trait HasTests extends FormatAssertions {
   def extension(filename: String): String = filename.replaceAll(".*\\.", "")
 
   def parseDiffTests(content: String, filename: String): Seq[DiffTest] = {
-    val sep =
+    val sep = {
```

Comment:
optional: i think the `{}` is not needed here

Reply:
Whoops, removed. Remnants from some debugging session. I don't mind enabling more aggressive Scalafmt options if you have preferences

Reply:
i do not, really. it's just our general approach to code changes is to reduce the amount of red and green, to make reviewing easier :)